Configure.pl

```diff
@@ -53,10 +53,15 @@
 
     unless (defined $options{prefix}) {
         my $default = defined($options{sysroot}) ? '/usr' : File::Spec->catdir(getcwd, 'install');
-        print "ATTENTION: no --prefix supplied, building and installing to $default\n";
+        # print "ATTENTION: no --prefix supplied, building and installing to $default\n";
```

Comment:
Why do you remove this message? Doesn't seem related to the other changes and the message is actually quite helpful.